tests/acceptance/features/webUIOauth2/enforceTokenAuth.feature

```diff
@@ -0,0 +1,52 @@
+@webUI @insulated @disablePreviews
+Feature: enforce token auth
+
+As an administrator
+I want to be able to enforce token based auth
+So that I can improve the security of the system by forbidding basic auth with username & password
+
+	Background:
+		Given these users have been created:
+			| username    | password | displayname  | email                 |
+			| user1       | 1234     | User One     | u1@oc.com.np          |
+		And token auth has been enforced
+		And the user "user1" has correctly established an oauth session
+
+	Scenario: access files app with oauth when token auth is enforced
+		When user "user1" requests "/index.php/apps/files" with "GET" using basic auth
+		Then the HTTP status code should be "401"
+		When the user requests "/index.php/apps/files" with "GET" using oauth
+		Then the HTTP status code should be "200"
+
+	Scenario: using WebDAV with oauth when token auth is enforced
+		When user "user0" requests "/remote.php/webdav" with "PROPFIND" using basic auth
```

Comment:
@individual-it why does this say `user0` ?
`user1` exists but `user0`  does not exist.